src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/lint/LintWorkerAction.kt

```diff
@@ -45,14 +46,14 @@ abstract class LintWorkerAction : WorkAction<LintWorkerParameters> {
                     return@forEach
                 }
 
-                ktLintEngine.lint(Code.CodeFile(file)) { error ->
+                ktlintEngine.lint(Code.fromFile(file)) { error: LintError ->
```

Comment:
To me this is a bit baffling. `lint` api calls return `LintError` which seems legacy, but we have to feed a `KtLintCliError`, which is slightly richer, into Reporters. The name almost sounds like it's being moved out of API consumer land?
In any case, made it work.